### PR TITLE
Improve `else` documentation for functions

### PR DESCRIPTION
**Pull request by lukyer**
Context: https://github.com/open-policy-agent/community/discussions/244

### CODE REVIEW

File: docs/content/policy-language.md [hunk 1]
```diff
@@ -975,6 +975,15 @@ s(5, 3)
 ```live:eg/double_function_define_undefined/2:output:expect_undefined
 ```
 
+Functions definition may contain `if/else` semantics similarly to [rules](#else-keyword):
+```
+can_edit(entity) := true if {
+	entity.owner == input.user
```
Comment (anderseknert) [inline docs/content/policy-language.md:981]: Indentation should look the same as the other examples here.

File: docs/content/policy-language.md [hunk 2]
```diff
@@ -975,6 +975,15 @@ s(5, 3)
 ```live:eg/double_function_define_undefined/2:output:expect_undefined
 ```
 
+Functions definition may contain `if/else` semantics similarly to [rules](#else-keyword):
```
Comment (anderseknert) [inline docs/content/policy-language.md:978]: Functions definition -> Function definitions

File: docs/content/policy-language.md [hunk 3]
```diff
@@ -975,6 +975,15 @@ s(5, 3)
...
+```
+can_edit(entity) := true if {
+	entity.owner == input.user
+} else := true {
```
Comment (anderseknert) [inline docs/content/policy-language.md:982]: Should this also be `true if`?

**Review by anderseknert — COMMENTED** [review]
Thanks for helping improve our docs! A few comments to consider, and you'll also need to add a sign-off for your commit:

```
git commit --amend --signoff
git push --force
```

File: docs/content/policy-language.md (hunk 3 above)
Comment (anderseknert) [reply docs/content/policy-language.md:982]: Oh, that's apparently not supported 😄 Ignore that for now then.

**Comment by srenatus** [comment]
🧹 Feel free to re-open when you find the time to get back to this 😃

**Pull request closed without merging**